packages/grid/x-data-grid/src/components/base/GridErrorHandler.tsx

```diff
@@ -38,20 +21,11 @@ function GridErrorHandler(props: { children: React.ReactNode }) {
       logger={logger}
       render={(errorProps) => (
         <GridMainContainer>
-          <GridAutoSizer
```

Comment:
I don't see any use case for having `GridAutoSizer` here.
It was added in https://github.com/mui/mui-x/pull/3558#discussion_r779934238, but I don't think it's necessary - here's the same demo with a package from this PR: https://codesandbox.io/s/datagriddemo-material-demo-forked-dzx7nl?file=/demo.tsx